### ISSUE TITLE
Files deleted on dropbox with copy command

### ISSUE THREAD

**Issue by chadyj** [ncw/rclone#474]
Loving rclone!  Great work on a genuinely useful app.

I'm copying 100gb from S3 to Dropbox and things were seemingly going well, but I received an email from Dropbox saying:

> We noticed that you recently deleted a large number of files from your Dropbox. Deleted files are saved for 30 days, and can be restored anytime during that 30-day window.

I checked and saw that thousands of files had been deleted. I searched the docs and couldn't find mention of any deletions during a copy command. 

Not sure if it is related but I'm also seeing hundreds of 16 byte files on Dropbox with the same name as the folder, eg:

folder-name (actual folder with files)
folder-name (1)
folder-name (2)
folder-name (3)

How can I debug further? And should I re-run the copy command to ensure all the files were transferred?

> What is your rclone version (eg output from `rclone -V`)

rclone v1.29

> Which OS you are using and how many bits (eg Windows 7, 64 bit)

Amazon Linux AMI release 2016.03

> Which cloud storage system are you using? (eg Google Drive)

S3 to Dropbox

> The command you were trying to run (eg `rclone copy /tmp remote:tmp`)

rclone copy sss:bucket/folder drop:folder

**Comment by ncw** [comment]
`rclone copy` does't delete files - you need `rclone sync` for that.

I wonder if you are running the dropbox desktop client elsewhere and it is deleting the files you are uploading? Or are you using some other tool simultaneously with rclone?

I don't know what the 16 byte files might be about either, not seen anything like that :-(

Can you run rclone with the `-v` flag and see if there is anything interesting in the log?

**Comment by chadyj** [comment]
> I wonder if you are running the dropbox desktop client elsewhere and it is deleting the files you are uploading? Or are you using some other tool simultaneously with clone?

Not using any other tools. 

> Can you run rclone with the -v flag and see if there is anything interesting in the log?

The 16 byte files still get created but nothing suspicious showing up in the log.

**Comment by ncw** [comment]
Can you send me the result of `rclone lsl s3:bucket` so I can see what the source looks like and `rclone lsl dropbox:destination` so I can see what the destination looks like.  Either post it here or send it privately to nick@craig-wood.com

Can you send me the log of the transfer too please (the one you made with `-v`).

Thanks

Nick

**Comment by ncw** [comment]
I'm going to close this as we haven't made any progress on it recently.  If you have any more ideas please re-open.  Thanks, Nick

**Comment by disimone** [comment]
Hi,

I stumbled on this old topic and I thought I'd ask here, since I am having the same problem. After I upgraded to v1.38, when I try to push some local files on dropbox (using rclone copy), the files on dropbox are deleted, then retransferred.

The previous transfer was using an older version (1.34, IIRC) and the dropbox API v1.

Running with the -v option does not really give more information as of why things are cancelled.

I am happy to help in further debugging, but I have no clue what to be looking for.

Thanks for your help,

Andrea.
